_includes/sxs/02-extending/part-two.html

```diff
@@ -50,19 +50,19 @@ <h3 class="step">Import the new stylesheet into USWDS custom styles.</h3>
   <div class="editor">
     <h4 class="filename"><span class="material-icons" aria-hidden="true">description</span> _theme/_uswds-theme-custom-styles.scss</h4>
 <pre class="blur">@use "uswds-core" as *;
-@forward "../assets/css/styles.scss";  
+@forward "../assets/css/styles.scss";
 </pre><pre class="add">@forward "../assets/css/_my-card.scss";</pre>
```

Comment:
Additional `_theme/components` change

```suggestion
</pre><pre class="add">@forward "../_theme/components/_my-card.scss";</pre>
```